src/App/ComponentsDocumentation/components/Buttons/__snapshots__/index.test.js.snap

```diff
@@ -987,14 +1054,6 @@ exports[`Documentation: Buttons Overview renders 1`] = `
                     "icon": "cloud",
                   },
                 },
-                Object {
```

Comment:
Shouldn't loader be available even though loader documentation was added? Might be good to give developers easy access to all the options available